src/main/scala/util/PlusArg.scala

```diff
@@ -83,7 +83,7 @@ object PlusArgArtefacts {
     * @param docstring text to include in the help
     * @param doctype description of the Verilog type of the PlusArg value (e.g. STRING, INT)
     */
-  def append[A](name: String, default: Option[A], docstring: String, doctype: String): Unit =
+  def append[A](name: String, default: Option[A], docstring: String, doctype: String = "INT"): Unit =
```

Comment:
I don't think that the `INT` default makes sense given the type of `A` is unknown.

Reply:
OK, fair enough. What about

```
def append(name: String, default: Option[BigInt], docstring: String): Unit =
  append[BigInt](name, default, docstring, "INT")
```

?